domains/aqua.json

```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "URL": "aqua.is-a.dev"
+        "URL": "aquaquokka.carrd.co"
```

Comment:
Please add `https://` in front of the URL.

Reply:
Ok

Reply:
@WilliamDavidHarrison Done.